apps/frontend/src/App.tsx

```diff
@@ -1,7 +1,36 @@
-import "./App.css";
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from "react-router-dom";
+import Login from "./pages/Login";
+import ForgotPassword from "./pages/ForgotPassword";
+import CreateAccount from "./pages/CreateAccount";
+import Homepage from "./pages/Homepage";
+import CreateListing from "./pages/CreateListing";
+import EditListing from "./pages/EditListing";
+import Account from "./pages/Account";
+import ViewListing from "./pages/ViewListing";
+import Messages from "./pages/Messages";
 
 function App() {
-  return <h1>Martletplace</h1>;
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Navigate to="/login" />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/forgot-password" element={<ForgotPassword />} />
+        <Route path="/create-account" element={<CreateAccount />} />
+        <Route path="/homepage" element={<Homepage />} />
+        <Route path="/create-listing" element={<CreateListing />} />
+        <Route path="/edit-listing" element={<EditListing />} />
+        <Route path="/account" element={<Account />} />
+        <Route path="/view-listing" element={<ViewListing />} />
+        <Route path="/messages" element={<Messages />} />
```

Comment:
Not sure if this has already been discussed but I would recommend creating the urls to be somewhat similar to the api endpoints for consistency. Perhaps something along the lines of:
```suggestion
        // If not logged in `<Navigate to="/login" />`
        <Route path="/" element={<Homepage />}

       // If the full URL contained the `:id` like `/user/:id` then it would show another users profile. If no `:id` then show the current users account.
        <Route path="/user" element={<Account />} /> 
        <Route path="/user/login" element={<Login />} />
        <Route path="/user/signup" element={<CreateAccount />} />
        <Route path="/user/reset" element={<ForgotPassword />} />

        <Route path="/listing/new" element={<CreateListing />} />
         // The full URL would be `/listing/edit/:id`
        <Route path="/listing/edit" element={<EditListing />} />
         // The full URL would be `/listing/view/:id`
        <Route path="/listing/view" element={<ViewListing />} />

        <Route path="/messages" element={<Messages />} />
```

Reply:
Thanks Max, addressed your comment!